packages/salesforcedx-apex-debugger/package.json

```diff
@@ -23,14 +23,14 @@
     "@salesforce/salesforcedx-test-utils-vscode": "58.16.0",
     "@types/async-lock": "0.0.20",
     "@types/chai": "4.3.3",
-    "@types/jest": "28.1.7",
+    "@types/jest": "^29.5.5",
```

Comment:
It's deps and dev deps.  I probably said that in a confusing way.  Previously the package-lock.json was already managing all the deps (dev and non-dev)  from all the packages, but each individual package got it's own copy of the dep in the node_modules folder.  Now with npm workspaces it still manages the lock file the same way, but there is only a single dep installed in the node_modules folder at the root instead of it being copied to all the various packages.  Def feels weird in that you look at a node_modules folder in a package and the dep isn't there. 
For example below in the utils package it depends on jest
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/76090802/4b795c9f-622a-4526-a5ce-b114640bc9d0)

But there is not jest folder in the package node_modules directory...only at the root 
![image](https://github.com/forcedotcom/salesforcedx-vscode/assets/76090802/4cba070d-f3e5-44a5-9205-1ef54487e15f)

Note these are all very under the hood changes.